test/json.50.lua

```diff
@@ -0,0 +1,147 @@
+--
```

Comment:
Out of scope for this PR, but we should thing about, rewriting this lib in TS and compiling it for the different version before injecting into the test runner.